1-exercises/D-array-filter/exercise.js

```diff
@@ -8,7 +8,8 @@
 
 let pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-let pairsByIndex; // Complete this statement
+let pairsByIndex = pairsByIndexRaw.filter(
+  (element) => !!element && element.length === 2)
 
 let students = ["Islam", "Lesley", "Harun", "Rukmini"];
```

Comment:
Hi Mehreen,  I was looking at this exercise while trying to fix mine and honestly, you make codes look beautiful. Question though, can you help me with the parsefloat please (not used here)? Looks like you really understand it and I just don't get it.... Well done superstar. xx